assets/js/modules/ads/utils/validation.js

```diff
@@ -27,5 +27,9 @@
  * @return {boolean} Whether or not the given ID is valid.
  */
 export function isValidAdsConversionID( value ) {
-	return typeof value === 'string' && /^AW-[0-9]+$/.test( value );
+	return (
+		typeof value === 'string' &&
+		/^AW-[0-9]+$/.test( value ) &&
+		value.length <= 20
```

Comment:
This doesn't seem necessary, it is just throwing the validation error for the valid length limit, we are already limiting it with `maxLength` prop in the field, which prevents number of character going over 20. Which can be valid characters, and in case there is a chance id reaching this number of characters we will invalidate it.

Reply:
Thanks @zutigrm, this was needed to align the placeholder in all cases but I take your point.